llvm/lib/Target/AArch64/AArch64LoadStoreOptimizer.cpp

```diff
@@ -1240,6 +1298,29 @@ AArch64LoadStoreOpt::mergePairedInsns(MachineBasicBlock::iterator I,
     MOp1.setReg(AArch64::Q0 + (MOp1.getReg() - AArch64::Z0));
     LLVM_DEBUG(((MachineInstr *)MIB)->print(dbgs()));
   } else {
+
+    // Restore the debug instruction numbers to the merged instruction.
+    bool UseNewDebugInstrNum = false;
+    unsigned NewDebugInstrNum;
```

Comment:
```suggestion
    unsigned DebugInstrNum = I->peekDebugInstrNum();
```
then you can skip the ternary operator later on.

Reply:
The number of times we're calling `peekDebugInstrNum` suggest there is something we can improve on